roles/workstation/files/xscreensaver-config

```diff
@@ -0,0 +1,256 @@
+# XScreenSaver Preferences File
```

Comment:
Could someone remind me why we are even checking in these files? It may be a good time to reevaluate if we actually need them